### DISCUSSION TITLE
Nginx Proxy manager update to 2.11.2

### DISCUSSION THREAD

**Discussion by aroundmyroom** [tteck/Proxmox#2970] [Help Needed]
Using nginx proxy manager from various versions before including v5 etc. and ran 2.11.1 (system is from end of 2022)
today tried to upgrade to 2.11.2 and it failed 

it crashed on a digital envelope error within webpack 
could not find the reason quickly cause normally the lxc update with Nginx Proxy Manager goes smooth

As I updated the lxc to bookworm to keep the lastest debian version myself it could be an issue with that, or it could be other dependencies.
After fiddling around I found out that by adding this environment variable

export NODE_OPTIONS=--openssl-legacy-provider
I could update to 2.11.2 without any further issues using

bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/nginxproxymanager.sh)"

it might be my issue, might occur to others. above could help others.
especially when the system is long time running and you made some changes to keep up OS version.

**Comment by nfribeiro** [comment]
Hi. I've tested and updated from version 2.11.1 and 2.11.2 and had no problems with the update.
But thanks for the warning
